lib/Cleantalk/Antispam/ProtectByShortcode.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Cleantalk\Antispam;
+
+use Cleantalk\ApbctWP\Variables\Server;
+
+class ProtectByShortcode
+{
+    public function __construct()
+    {
+        add_filter('ct_wordpress_protect_from_spam', array($this, 'protectByShortcode'), 10, 2);
+    }
+
+    public function protectByShortcode($data, $options = [])
+    {
+        $output = [
+            'is_spam' => false,
+            'message' => '',
+        ];
+
+        $input_array = apply_filters('apbct__filter_post', $data);
+        $data = ct_gfa($input_array);
+
+        $base_call_data = array(
```

Comment:
Do we need to sanitize data? I suppose if we provide a hook, we do need sanitize the data, otherwise there could be injections or invalid formats that breaks moderate response.